powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/AdditionalInformation.java

```diff
@@ -24,24 +24,63 @@
  */
 public class AdditionalInformation {
 
-    public static final String BLOCKED_REASON = "BLOCKED_REASON";
-    public static final String VAULT_UNLOCKED_REASON = "VAULT_UNLOCKED_REASON";
-    public static final String BLOCKED_REASON_MAX_FAILED_ATTEMPTS = "MAX_FAILED_ATTEMPTS";
-    public static final String BLOCKED_REASON_NOT_SPECIFIED = "NOT_SPECIFIED";
-    public static final String VAULT_UNLOCKED_REASON_NOT_SPECIFIED = "NOT_SPECIFIED";
-    public static final String BIOMETRY_ALLOWED = "BIOMETRY_ALLOWED";
-    public static final String ACTIVATION_OTP_FAILED_ATTEMPT = "OTP_FAILED_ATTEMPT";
-    public static final String ACTIVATION_OTP_MAX_FAILED_ATTEMPTS = "OTP_MAX_FAILED_ATTEMPTS";
-    public static final String ACTIVATION_OTP_VALUE_UPDATE = "OTP_VALUE_UPDATE";
-
-    /**
-     * For activation history audit.
-     * <ul>
-     * <li>during activation commit ({@code null} version -&gt; specific version)</li>
-     * <li>during upgrade commit (version changes to a newer one)</li>
-     * </ul>
-     */
-    public static final String ACTIVATION_VERSION_CHANGED = "ACTIVATION_VERSION_CHANGED";
+    public static class Key {
+        
+        /**
+         * Key describing reasons for blocked activation.
+         */
+        public static final String BLOCKED_REASON = "BLOCKED_REASON";
```

Comment:
When refactoring, can we change it to enum (maybe in s separate PR)?

Reply:
Let's do this in a separate issue. The benefit of the change is minimal.